stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{token}').onWrite(event => {
+  const token = event.data.val();
+  const user = event.params.userId;
```

Comment:
that isn't going to be the stripe customer id but instead the user id in firebase, but as I mentioned above this schema change conflicts with how the customer_id is currently stored